includes/utils.php

```diff
@@ -11,33 +11,33 @@
 
 class WPBDP__Utils {
 
-    /**
-     * @since 5.2.1
-     */
-    public static $property = null;
-
-    /**
-     * @since 3.6.10
-     */
-    public static function normalize( $val = '', $opts = array() ) {
-        $res = strtolower( $val );
-        $res = remove_accents( $res );
-        $res = preg_replace( '/\s+/', '_', $res );
-        $res = preg_replace( '/[^a-zA-Z0-9_-]+/', '', $res );
-
-        return $res;
-    }
-
-    /**
-     * @since 5.0
-     */
-    public static function sort_by_property( &$array, $prop ) {
-        self::$property = $prop;
-
-        uasort( $array, array( 'WPBDP_Utils', 'sort_by_property_callback' ) );
-
-        self::$property = null;
-    }
+	/**
```

Comment:
This diff is too big. Can you please back out the phpcs for lines that aren't being changed?

Reply:
This has been fixed :)